style.css

```diff
@@ -371,7 +375,7 @@ a {
   background: #fff;
   border: 1px solid #cfd8dc;
   border-radius: 8px;
-  width: 327px;
+  width: 355px;
   height: 48px;
-  margin-left: 20px;
+  margin-left: 15px;
```

Comment:
- [ ] As suggested by the previous reviewer and stated in the project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly adjust the section background covers the entire width of your screen as per the Figma design below:

The Figma design Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/207336010-774d216e-366c-45d1-95a9-794b32effd01.PNG)

Your Project Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/207336648-da465666-6169-404f-9068-e2d685a6fab8.PNG)

- [ Optional ] Kindly as stated the project requirement I recommend you to  Implement the button interactions (enable, hover, etc.) Just Like the Figma design.

![Capture](https://user-images.githubusercontent.com/93598189/207338320-f2037a84-b3b2-431e-9b1f-d17f0b64b083.PNG)